front/src/pages/Home/Home.tsx

```diff
@@ -18,12 +19,23 @@ const Home = () => {
     isFetching,
   } = useFetchTodos();
   const { mutate: updateTodo } = useUpdateTodo();
+  const { mutate: deleteTodo } = useDeleteTodo();
   const navigate = useNavigate()
 
   const handleTodoClick = (id: string, isDone: boolean) => {
     updateTodo({ id, isDone });
   }
 
+  const handleEditButtonClick = (ev: React.MouseEvent<HTMLButtonElement>, todoId: string) => {
+    ev.stopPropagation();
```

Comment:
and why we need `stopPropagation()` here? what is other handler upper in DOM?

Reply:
@skyboyer 
It is on line 20 - handleTodoClick().
If we don't use stopPropagation() then "isDone" prop will be changed.